### PR TITLE
#121/1 hall of fame content

### PR DESCRIPTION
**Pull request by RobertS-ki**
Testing branch renamed to subbranch of issue #121. Fixed/added function/object parameters: mode and level, that are passed thru objects and button in order to finally allow hall of fame window to adapt its content to level and (game) mode change. Hall of fame content is still not rendered (only css styled mockup). Game difficulty level is based on const.js constant.

### CODE REVIEW

File: tests/localStorage/przykladowyranking.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    {
+      name: 'Test Name',
+      result: 500,
+    },
+    {
+      name: 'Test Name',
+      result: 400,
+    },
+  ],
+  houses_hard: [
```
Comment (vieraboschkova) [inline tests/localStorage/przykladowyranking.js:72]: Great, that is exactly the kind of mock that is needed if you need to prepare UI for a functionality that is a work in progress. Good job!

**Comment by RobertS-ki** [comment]
Working Hall of fame button and Hall of fame content window. Button toggles to Rules button, content follow the toggle.
Css styles updated and unified css between icons on buttons and in Moder Rules window. Code cleaned (console.log, unneccessary comments, etc). Merged with latest development.

**Review by n0macx — COMMENTED** [review]
Almost Nice. Very Nice. Good job!

**Review by Smyku6 — APPROVED** [review]

**Review by MonikaKrella — APPROVED** [review]
good job men

**Pull request merged**